.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,3 @@
-## Issue [INNO-XXXX](https://webgate.ec.europa.eu/CITnet/jira/browse/INNO-XXXX)
+## Issue
```

Comment:
I'm not confident with PULL_REQUEST_TEMPLATE.md, maybe @degliwe could share his thoughts